code/datums/components/radioactive.dm

```diff
@@ -76,6 +76,8 @@
 /datum/component/radioactive/proc/rad_attack(datum/source, atom/movable/target, mob/living/user)
 	radiation_pulse(parent, strength/20)
 	target.rad_act(strength/2)
+	if(!hl3_release_date)
```

Comment:
this isn't funny

Reply:
well I mean it is, but what is it?

Reply:
It's part of the radioactive component to control half life